ros_gz_sim_demos/launch/robot_description_publisher.launch.py

```diff
@@ -76,14 +76,21 @@ def generate_launch_description():
     )
 
     # Spawn
-    spawn = Node(package='ros_gz_sim', executable='create',
-                 arguments=[
-                    '-name', 'my_custom_model',
-                    '-x', '1.2',
-                    '-z', '2.3',
-                    '-Y', '3.4',
-                    '-topic', '/robot_description'],
-                 output='screen')
+    spawn = Node(
+        package='ros_gz_sim',
+        executable='create',
+        parameters=[
+            {
+                'topic': '/robot_description'
+            }
+        ],
+        arguments=[
+            '-name', 'my_custom_model',
+            '-x', '1.2',
+            '-y', '3.3',
+            '-z', '2.4'
```

Comment:
these arguments are parameters now, aren't they ?

Reply:
> these arguments are parameters now, aren't they ?

Yes, I'll move them to parameters.